src/org/labkey/test/tests/SampleTypeLinkedStudyExportTest.java

```diff
@@ -95,6 +95,13 @@ public void testImportSampleTypeFolder()
                 "Date", now,
                 "ParticipantID", "P1"));
 
+        samplesTable = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+        samplesTable.clickImportBulkData();
+        String header = "AliquotedFrom\n";
+        String aliquotRow =  "Blood\n";
+        setFormElement(Locator.name("text"), header + aliquotRow);
+        clickButton("Submit");
```

Comment:
@XingY FYI, `clickImportBulkData` returns a page object that has methods for setting that input and submitting. Using that would be preferable to having the locator and button text defined in this test. Using existing components and page classes gives us one place to update if/when the import page get redesigned at some point.